source/simulator/parameters.cc

```diff
@@ -611,6 +611,14 @@ namespace aspect
                          "as opposed to continuous. This then requires the assembly of face terms "
                          "between cells, and weak imposition of boundary terms for the composition "
                          "field via the discontinuous Galerkin method.");
+      prm.declare_entry ("Use limiter for discontinuous solution", "false",
+                         Patterns::Bool (),
+                         "Whether to apply the bound preserving limiter as a postprocess after having"
+                         "the discontinous solution. Currently we should apply this only to the"
+                         "compositional solution if the global max/min are already determined and defined"
+                         "This limiter will keep the discontinuous solution in the range given by global max/min."
```

Comment:
Please specify the last point a bit more. "global max/min" could mean the global maximum and minimum in the solution, but you mean "a user defined maximum and mimum value".